config/server/MarioQuartz.cfg

```diff
@@ -0,0 +1,17 @@
+{
+    dir = "",
+    name = "InputRaylib",
+    args = {}
+},
+{
+    dir = "",
+    name = "MoveRaylib",
+    args = {}
+},
+{
+    dir = "",
+    name = "Config",
+    args = {
+        path = "./config/levels/config2.cfg"
+    }
+}
```

Comment:
Does the mario quartz really only needs those systems ?

Reply:
Why would it need anything more as of right now ?

Reply:
I don't know it seems really minimalist compared to the config file of the r-type. Shouldn't there be a move system ? Or is it the MoveRaylib that does it ? In this case maybe it should not be named with raylib in it.